src/XrdHttp/XrdHttpReq.hh

```diff
@@ -43,6 +43,7 @@
 
 #include "XProtocol/XProtocol.hh"
 #include "XrdXrootd/XrdXrootdBridge.hh"
+#include "XrdHttpChecksumHandler.hh"
 
 #include <vector>
```

Comment:
OK, I get it. the header inversion was there before you started. However, we might as well correct it in the files that git updated. We should make note that it should be corrected everywhere: system header followed by local headers.